client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -157,15 +159,15 @@ public void testCreateIndex() throws IOException {
 
             // tag::create-index-request-mappings
             request.mapping("tweet", // <1>
-                "  {\n" +
-                "    \"tweet\": {\n" +
-                "      \"properties\": {\n" +
-                "        \"message\": {\n" +
-                "          \"type\": \"text\"\n" +
-                "        }\n" +
-                "      }\n" +
-                "    }\n" +
-                "  }", // <2>
+                "{\n" +
+                    "  \"tweet\": {\n" +
+                    "    \"properties\": {\n" +
+                    "      \"message\": {\n" +
+                    "        \"type\": \"text\"\n" +
+                    "      }\n" +
+                    "    }\n" +
+                    "  }\n" +
+                    "}", // <2>
```

Comment:
was this required? Maybe you can revert this small part?

Reply:
The IDE indented the lines; will revert.

Would you like me to add the two extra spaces per line within the string back in? They seemed superfluous.